### DISCUSSION TITLE
scoop streamlink is out-dated (5.1.2.-1 currently)

### DISCUSSION THREAD

**Discussion by Carl-Robert** [streamlink/streamlink#5253] [General]
### Description

Hey!

The scoop repository installs 5.1.2-1 version which is quite out-dated. Could you investigate and fix the scoop version?

**Comment by bastimeyer** [comment]
This is third party packaging. Ask the scoop maintainers. We can't do anything here.

**Reply by bastimeyer** [reply]
https://github.com/ScoopInstaller/Extras/issues

Or submit a pull request with an update yourself:
https://github.com/ScoopInstaller/Extras/blob/master/bucket/streamlink.json

**Reply by Carl-Robert** [reply]
Sure thing! Wasn't sure who was maintaining it. I'll raise an issue on the right tracker.
